nuancier/forms.py

```diff
@@ -48,7 +52,39 @@ def is_number(form, field):
         raise wtf.ValidationError('Field must contain a number')
 
 
-class AddElectionForm(flask_wtf.Form):
+class BaseForm(flask_wtf.Form):
+    """
+    Provide a base form class.
+
+    For new versions of flask-wtf (greater than 0.10.0), this behaves exactly
+    like :class:`flask_wtf.Form`.
+
+    Versions of flask-wtf prior to 0.10.0 did not have or did not properly use
+    the 'WTF_CSRF_TIME_LIMIT' setting. This configures the self.TIME_LIMIT
+    value used by flask-wtf if the delta is not None and it's an old version of
+    flask-wtf. See https://github.com/lepture/flask-wtf/issues/131 for more
+    information.
+    """
+
+    def __init__(self, *args, **kwargs):
+        delta = APP.config.get('WTF_CSRF_TIME_LIMIT', 3600)
```

Comment:
I'm seeing that I do the same in pagure and looking at the sources of flask_wtf, looks like we should be safe indeed.